internal/cloudrunci/cloudrunci.go

```diff
@@ -259,6 +265,7 @@ func (s *Service) operationLabel(op string) string {
 func (s *Service) deployCmd() *exec.Cmd {
 	args := append([]string{
 		"--quiet",
+		"alpha", // TODO until --use-http2 goes GA
```

Comment:
I didn't realize the alpha component was installed, nor do I see it installed in the code, but since their are other, not-skipped tests that use `gcloud alpha` it must be fine.

Reply:
I suspect --quiet accepts installation if it's not installed.